src/vk_mem_alloc.h

```diff
@@ -7326,19 +7329,18 @@ class VmaRecorder
     {
     public:
         UserDataString(VmaAllocationCreateFlags allocFlags, const void* pUserData);
-        const char* GetString() const { return m_Str; }
+        const char* GetString() const { return m_Str.c_str(); }
 
     private:
         char m_PtrStr[17];
-        const char* m_Str;
+        std::string m_Str;
```

Comment:
Do we really need to use `std::string` even on Windows? This class is very slow, it does dynamic memory allocation.

Reply:
I will change it back.